src/utils.jl

```diff
@@ -214,3 +214,40 @@ macro matchable(expr)
         Base.length(x::$name) = $(length(fields) + 1)
     end |> esc
 end
+
+# Check for linear coefficients
+coefficients(expr,sym::SymbolicUtils.Sym) coefficients(expr,[sym])
```

Comment:
The signature should be `coefficients(f, expr)`

Reply:
Seems like wrong syntax too.